packages/bundler-plugin-core/src/types.ts

```diff
@@ -97,6 +97,20 @@ export interface Options {
    * Example: `rollup-package`
    */
   bundleName?: string;
+
+  sentryOptions: {
+    /** Only send bundle stats to sentry (used within sentry webpack plugin). */
+    sentryOnly?: boolean;
+
+    /** The token to use for uploading the bundle analysis information to sentry. */
+    sentryAuthToken?: string;
+
+    /** The name of the sentry organization to send webpack stats to. */
```

Comment:
this works for more than just webpack, so we can make it more generic.

Reply:
Yeah good call!, i'll update the comments